lord/time_lord_test.go

```diff
@@ -342,6 +410,11 @@ var _ = DescribeTable("A range with an interval and a previous time", (testCase)
 		nowDay:  time.Thursday,
 
 		result: true,
+		latest: expectedTime{hour: 13, weekday: time.Thursday},
+		list: []expectedTime{
+			{hour: 14, minute: 58, weekday: time.Wednesday},
```

Comment:
Note to self: stop is exclusive, so it is correct for this to be the last version.